### PR TITLE
Add more timelines and fix user timeline

### LINKED ISSUE #10: Timelines
All timelines use the same template at https://github.com/Herover/itu-devops-h/blob/dev/minitwit-legacy/templates/timeline.html
https://github.com/Herover/itu-devops-h/blob/dev/minitwit-legacy/minitwit.py#L92
https://github.com/Herover/itu-devops-h/blob/dev/minitwit-legacy/minitwit.py#L112
https://github.com/Herover/itu-devops-h/blob/dev/minitwit-legacy/minitwit.py#L121

### PR DESCRIPTION
**Pull request by Herover**
Closes #10 , gravatar and time formater not implemented.

### CODE REVIEW

File: minitwit/src/main/java/WebApplication.java [hunk 1]
```diff
@@ -85,22 +89,147 @@ public static String render(Map<String, Object> model, String templatePath) {
...
+            if (profileRs.isClosed()) {
+                response.status(404);
+                return "404"; // TODO: What does the old one do?
+            }
+            var profileUser = new HashMap<String, Object>();
+            profileUser.put("user_id", profileRs.getInt("user_id"));
+            profileUser.put("username", profileRs.getString("username"));
+            profileUser.put("email", profileRs.getString("email"));
+            model.put("profile_user", profileUser);
+            profileRs.close();
```
Comment (pizzaluc) [inline minitwit/src/main/java/WebApplication.java:203]: We might wanna refactor this out into another function, as i expect many of the functions need to return the user from a username.

File: minitwit/src/main/java/WebApplication.java [hunk 2]
```diff
@@ -85,22 +89,147 @@ public static String render(Map<String, Object> model, String templatePath) {
     }
 
     public static Route servePublicTimelinePage = (Request request, Response response) -> {
+        try {
```
Comment (pizzaluc) [inline minitwit/src/main/java/WebApplication.java:92]: Maybe its not something we should worry about now, but I have seen that basically all our functions just have a giant try catch over every line of code, seems a bit sketchy

**Review by pizzaluc — APPROVED** [review]
Looks good, I might refactor one of the pieces of code out, so reuse it in some other endpoints, but other than that it looks clean

**Review by KnittedSox — APPROVED** [review]
Seems solid

File: minitwit/src/main/java/WebApplication.java (hunk 2 above)
Comment (Herover) [reply minitwit/src/main/java/WebApplication.java:92]: I agree, but without it all exceptions are caught by a generic code 500 page that doesn't tell anything about what happened anywhere. Maybe we could make some sort of wrapper around the server functions so its the same...

**Pull request merged**